doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -85,13 +85,15 @@ SUIT DFU for nRF54H20
 Nordic Secure Immutable Bootloader (NSIB, B0, or B0n)
 -----------------------------------------------------
 
-Custom printing has been dropped in favor of using the logging subsystem, with output printed out to the default logging device.
-The ``CONFIG_SECURE_BOOT_DEBUG`` Kconfig option has been removed.
-To disable logging in B0 or B0n, set the :kconfig:option:`CONFIG_LOG` option to ``n``.
-To send logs over RTT instead of UART, apply the following settings:
+.. toggle::
+
+   Custom printing has been dropped in favor of using the logging subsystem, with output printed out to the default logging device.
```

Comment:
```suggestion
   Custom printing is replaced by the logging subsystem, with output printed out to the default logging device.
```

Reply:
This is not something I wrote. All I did here was add the toggle...

Reply:
This was reviewed and approved by Anna in https://github.com/nrfconnect/sdk-nrf/pull/17079